src/server/endpoint.cc

```diff
@@ -14,6 +14,7 @@ namespace Http {
 
 Endpoint::Options::Options()
     : threads_(1)
+    , maxPayload_(Const::DefaultMaxPayload)
```

Comment:
I thought that we submitted a PR a few weeks ago to fully initialize ALL members of Endpoint::Options.  Oh well.

Igor, I know that this request is slightly beyond the scope of this PR, but please init all 4 members to sane defaults, and in the same order that they are declare in the struct (or "-Weffc++" and cppcheck will complain about the init order).

Reply:
@dennisjenkins75 I have done it. Now I'm exploring the situation with `http_client_test` test and I have already found the root of problem. Hope in the next PR I will fix it.